Snake.fs

```diff
@@ -21,18 +21,27 @@ module SnakeGame =
         | Tail
 
     /// 🕹️ Our basic runtime information:
-    and Game = 
+    and Game =
         | GameRunning of Movement * Snake<Position> * Grid * Eaten:int * Food
         | GameOver of Grid * Eaten:int
 
     /// 🧭 x & y in our plane:
     and Position = int * int
 
-    /// 🍎 The food our snake will eat:
-    and Food = Position * string
-    
+    /// 🍎 The food sprite our snake will eat:
+    and Food = Position * FoodType
+    ///    food can be of various types (this is just used to change how it's displayed)
+    and FoodType = int
+
+    /// Contents of our Grid
+    and Cell =
+        | SnakeHead
+        | SnakeBelly
```

Comment:
looks like a half-finished change, this'd probably be more usefully aligned to its domain:

```fsharp
and Cell = 
   | Occupied                  // something we can't move into, game over!
   | Food of FoodType    // something we can eat, happy snake!
   | Empty                       // somewhere we can move 
```